site.config.js

```diff
@@ -0,0 +1,32 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
+import 'terra-core/packages/terra-site/src/cerner-mock-theme.scss';
+
+import navConfig from './navigation.config';
+// import componentConfig from './generatedComponentConfig';
+import componentConfig from './componentConfig';
+
+const siteConfig = {
+  /* The navigation configuration.  */
+  navConfig,
+
+  /* The component configuration path. */
+  componentConfig,
+
+  /* The logo to display as the placeholder. */
+  placeholderSrc: 'https://github.com/cerner/terra-core/raw/master/terra.png',
```

Comment:
It’d be nice to replace this with an svg at some point. Not sure the exact scope needed, as we may need UX help in it possibly.